src/github/middleware.ts

```diff
@@ -151,15 +155,7 @@ export default (
 			}
 			const util = getJiraUtil(jiraClient);
 
-			try {
-				context.sentry.captureMessage(
-					`Middleware: webhook handler - context: ${context}, jiraClient: ${jiraClient}, util: ${util}`
-				);
-				return await callback(context, jiraClient, util);
-			} catch (err) {
-				context.sentry.captureException(err);
-				throw err;
-			}
+			return callback(context, jiraClient, util);
```

Comment:
Have you tested this? Do we need the await at all?

Reply:
the linter told me to remove the `await`, because it's not needed in return statements. Which, thinking about it, is not a good linting rule, because it's dangerous for refactorings. I'll re-add the `await`